src/data/get_data.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
```

Comment:
I’d like to make this all python... 

datetime.datetime(2012,4,1,0,0).timestamp()
1333234800.0